virttest/qemu_monitor.py

```diff
@@ -2276,9 +2276,12 @@ def balloon(self, size):
         # Send a balloon monitor command
         self.send_args_cmd("%s value=%s" % (cmd, size))
         # Look for BALLOON QMP events
-        if not utils_misc.wait_for(lambda: self.get_event(qmp_event), 120):
-            raise QMPEventError(cmd, qmp_event, self.name)
-        logging.info("%s QMP event received" % qmp_event)
+        if self.verify_status('running'):
```

Comment:
With my comment in https://github.com/autotest/tp-qemu/pull/840, i think it's better to keep original. We need to confirm "No balloon change event received" before we change vm's status to "running" in case "balloon guest when it starts with paused status".